src/App.tsx

```diff
@@ -110,24 +123,38 @@ const App: React.FC = () => (
   <>
     <GlobalStyles />
     <Router basename={process.env.BASE_URL}>
-      <SwapLayout>
-        <React.Suspense fallback={null}>
-          <Switch>
-            <PrivateRoute path="/orders" exact component={Orders} />
-            <Route path="/trade/:buy-:sell" component={Trade} />
-            <PrivateRoute path="/liquidity" exact component={Strategies} />
-            <PrivateRoute path="/wallet" exact component={Wallet} />
-            <Route path="/about" exact component={About} />
-            <Route path="/faq" exact component={FAQ} />
-            <Route path="/book" exact component={OrderBook} />
-            <Route path="/connect-wallet" exact component={ConnectWallet} />
-            <Route path="/trades" exact component={Trades} />
-            <Route path="/settings" exact component={Settings} />
-            <Redirect from="/" to={initialUrl} />
-            <Route component={NotFound} />
-          </Switch>
-        </React.Suspense>
-      </SwapLayout>
+      <Switch>
+        <Route path="/v2">
+          <TradingLayout>
+            <React.Suspense fallback={null}>
+              <Switch>
+                <Route path="" exact component={Trading} />
```

Comment:
```suggestion
                <Route path="/v2" exact component={Trading} />
```
there are no relative routes in react-router v5, this one becomes a catch-all with `path=""`
You can look forward to [v6](https://reacttraining.com/blog/react-router-v6-pre/#relative-route-path-and-link-to) though